src/app/tracking/tracking.page.ts

```diff
@@ -18,6 +24,12 @@ export class TrackingPage implements OnInit, OnDestroy {
   @Input() stateIcon: string
   @Input() notificationsEnabled: boolean
   trajectoryExists: boolean
+  consented: boolean
+  firstTime: boolean
+
+  consent: boolean
+  informedConsent: InformedConsent
+  firstTimeConsent: FirstTimeConsent
```

Comment:
Consider more expressive names for this variables. For example – just from looking at this – I couldn't tell what `firstTime` actually means/does in the context of this component.